pkg/planner/core/casetest/plan_test.go

```diff
@@ -36,10 +36,7 @@ func getPlanRows(planStr string) []string {
 }
 
 func compareStringSlice(t *testing.T, ss1, ss2 []string) {
-	require.Equal(t, len(ss1), len(ss2))
-	for i, s := range ss1 {
-		require.Equal(t, len(s), len(ss2[i]))
-	}
+	require.Equal(t, ss1, ss2)
```

Comment:
how about ss1 is shaow cp of ss2?

Reply:
Revert this change; it's not crucial and it affects the cherrypick process.